### PR TITLE
Cleanup: replace some hardcoded vars and remove unused functions

### PR DESCRIPTION
**Pull request by NickrenREN**
Replace some hardcoded vars and remove unused functions

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: test/e2e/daemon_set.go [hunk 1]
```diff
@@ -94,8 +94,8 @@ var _ = framework.KubeDescribe("Daemon set [Serial]", func() {
 
 	f = framework.NewDefaultFramework("daemonsets")
 
-	image := "gcr.io/google_containers/serve_hostname:v1.4"
-	redisImage := "gcr.io/k8s-testimages/redis:e2e"
+	image := framework.ServeHostnameImage
+	redisImage := redisImage
```
Comment (xiangpengzhao) [inline test/e2e/daemon_set.go]: Why do we need this line?

File: test/e2e/resize_nodes.go [hunk 2]
```diff
@@ -32,13 +32,11 @@ import (
...
-	resizeNodeNotReadyTimeout = 2 * time.Minute
-	nodeReadinessTimeout      = 3 * time.Minute
-	podNotReadyTimeout        = 1 * time.Minute
-	podReadyTimeout           = 2 * time.Minute
-	testPort                  = 9376
+	resizeNodeReadyTimeout = 2 * time.Minute
+	nodeReadinessTimeout   = 3 * time.Minute
+	podNotReadyTimeout     = 1 * time.Minute
+	podReadyTimeout        = 2 * time.Minute
+	testPort               = 9376
```
Comment (xiangpengzhao) [inline test/e2e/resize_nodes.go]: Do we need gofmt in our e2e code?

File: test/e2e/daemon_set.go (hunk 1 above)
Comment (NickrenREN) [inline test/e2e/daemon_set.go]: redisImage is var defined in another file

File: test/e2e/resize_nodes.go (hunk 2 above)
Comment (NickrenREN) [inline test/e2e/resize_nodes.go]: Already run

File: test/e2e/daemon_set.go (hunk 1 above)
Comment (xiangpengzhao) [inline test/e2e/daemon_set.go]: then we can use that var directly, right?

File: test/e2e/resize_nodes.go (hunk 2 above)
Comment (xiangpengzhao) [inline test/e2e/resize_nodes.go]: sorry, it just seems not gofmt'd :)

File: test/e2e/daemon_set.go (hunk 1 above)
Comment (NickrenREN) [inline test/e2e/daemon_set.go]: Oh, my mistake,you are right. I will modify it tomorrow

Comment (NickrenREN) [inline test/e2e/daemon_set.go]: Addressed @xiangpengzhao  Thanks

**Comment by NickrenREN** [comment]
@rmmh Just rebase the codes and address the comment(delete redundant var)

**Pull request merged**